Lib/datetime.py

```diff
@@ -1095,7 +1160,7 @@ def isocalendar(self):
 
     def _getstate(self):
         yhi, ylo = divmod(self._year, 256)
-        return bytes([yhi, ylo, self._month, self._day]),
+        return bytes([yhi, ylo, self._month, self._day])
```

Comment:
The trailing comma here was not a mistake, and removing it changes the behavior. `__reduce__`'s second value in the returned `tuple` is required to be a `tuple` itself; by removing the comma, you change it from a one-`tuple` containing a `bytes` object to just a `bytes` object. This violates the pickle protocol, and even if it works (there are some weird hacks in the CPython C API that apply when a function that's supposed to accept a `tuple` of `args` will wrap a non-`tuple` into a one-`tuple`), it's not something you can or should rely on (PyPy and other interpreters use the Python-level code as is, and likely lack the same hacks).

Reply:
Oops. As color changed in Sublime, I removed. I forgot that one should not remove trailing comma for single element tuple.